nextflow_schema.json

```diff
@@ -12,17 +12,36 @@
+                    "schema": "assets/schema_phyloplace_input.json",
                     "mimetype": "text/csv",
                     "pattern": "^\\S+\\.csv$",
-                    "description": "Path to comma-separated file containing information about the samples in the experiment.",
-                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with at least five columns, and a header row. See [usage docs](https://nf-co.re/phyloplace/usage#samplesheet-input).",
+                    "description": "Path to comma-separated file containing information about phylogenetic placements to perform.",
+                    "help_text": "This specifies parameters for direct phylogenetic placement of query sequences onto a reference phylogeny, in contrast to when a HMMER search is first performed, see `--phylosearch_input`. It has to be a comma-separated file with at least five columns, and a header row ('sample', 'queryseqfile', 'refseqfile', 'refphylogeny' and 'model'). See [usage docs](https://nf-co.re/phyloplace/usage#samplesheet-input).",
                     "fa_icon": "fas fa-file-csv"
                 },
+                "phylosearch_input": {
+                    "type": "string",
+                    "format": "file-path",
+                    "exists": true,
+                    "schema": "assets/schema_phylosearch_input.json",
+                    "mimetype": "text/csv",
+                    "pattern": "^\\S+\\.csv$",
+                    "description": "Path to comma-separated file containing information about phylogenetic searches to perform.",
+                    "help_text": "This specifies parameters for phylogenetic placement of unknown sequences onto reference phylogeny after first using HMMER to search through a collection of uknown sequences (see `--search_fasta`) in contrast to when only placing known query sequences, see `--phyloplace_input`. It has to be a comma-separated file with at least two columns that specifies search HMM profiles ('target' and 'hmm') and at least three columns specifying the reference phylogenies ('refseqfile', 'refphylogeny' and 'model') for placement of search results. (Decoy profiles, i.e. profiles only used for searches, not placements, are allowed.) See [usage docs](https://nf-co.re/phyloplace/usage#samplesheet-input).",
+                    "fa_icon": "fas fa-file-csv"
+                },
+                "search_fasta": {
+                    "type": "string",
+                    "format": "file-path",
+                    "mimetype": "text/plain",
+                    "description": "Fasta file with sequences to be searched with HMM profiles.",
+                    "help_text": "Sequences from this file matching an HMM profile, will be placed in the reference phylogeny corresponding to the HMM profile.",
```

Comment:
In case it isnt explained anywhere else, how does that work? Ids of HMMs must correspond to header of fasta? Completely? Partially? Just asking because that might cause me trouble...

Reply:
No, no name matching at all. What happens, and I hope this is -- otherwise it will be -- explained in the docs. All sequences in this file are matched against all profiles and placed in the phylogeny that corresponds to the best matching profile. If this was an ASV set we might have a profile for bacteria and one for archaea. After matching the profiles, those scoring best for bacteria will be placed in the bacterial tree and vice versa. This is what Danilo started to add to Ampliseq: All ASV sequences will be sent to hmmsearch, scored and the placed in one tree.